sale_credit_point/models/partner.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2018-2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, exceptions, fields, models
+
+POINT_OPERATIONS = [
```

Comment:
not an expert but to avoid one import, this can be added to the pointHistory model ? 
No blocking comment

Reply:
Thanks for spotting it. The PR has been updated.